internal/sourcecode/templates/laravel/octane/docker/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+
+# Start Octane with the roadrunner binary if we find
```

Comment:
If we deleted roadrunner in the Dockerfile, will it be found here?

Reply:
The deletion you mentioned prior was just in case, as I had cases where the `rr` binary on my macos installed made its way into the docker container. I believe that was when `.dockerignore` wasn't added before I updated `go:embed`.

The get-binary command called in the container adds the `rr` binary back in that's appropriate for the OS type (ARM vs AMD, etc).

(This'll get refactored a but this concept will likely remain).